scripts/ralph-loop.sh

```
@@ -0,0 +1,1057 @@
+
+  log "  Testing migrations on temporary Neon branch '${temp_branch}'..."
+
+  # Create temporary branch from experimental-v2
+  "$NEONCTL" branches create \
+    --project-id "$NEON_PROJECT_ID" \
+    --org-id "$NEON_ORG_ID" \
+    --parent "$NEON_PARENT_BRANCH_ID" \
+    --name "$temp_branch" \
+    --output json >/dev/null 2>&1
+
+  if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)